jbpm-wb-human-tasks/jbpm-wb-human-tasks-client/src/main/java/org/jbpm/workbench/ht/client/editors/taskslist/TaskAdminListViewImpl.java

```diff
@@ -41,5 +72,126 @@ public void initDefaultFilters() {
     public String getGridGlobalPreferencesKey() {
         return DATA_SET_TASK_LIST_PREFIX;
     }
+    
+    @Override
+    public void initColumns(ExtendedPagedTable<TaskSummary> extendedPagedTable) {
+        initCellPreview(extendedPagedTable);
```

Comment:
ideally, this method should be shared with the parent class to avoid duplication. only the error column is new here.